tools/release-plugin.sh

```diff
@@ -173,13 +173,13 @@ for PLUGIN in "${!PROJECTS[@]}"; do
 
 	RELEASED_PLUGINS=$(
 		jq \
-			--arg property "$( echo "$PLUGIN" | sed 's/\//_/' )_release_type" \
+			--arg property "$( echo "$PLUGIN" | sed 's/[\/\-]/_/g' )_release_type" \
```

Comment:
Bash can do this substitution.
```suggestion
			--arg property "${PLUGIN//[\/-]/_}_release_type" \
```